src/SportsHub.Business.Tests/Services/SubcategoryServiceTests.cs

```diff
@@ -42,7 +43,7 @@ public async Task GetSubcategoryByIdAsync_WhenIdIsValid_ReturnsExpectedSubcatego
             // Arrange
             var expectedSubcategoryId = Guid.NewGuid();
 
-            var expectedSubcategory = new Subcategory(name: "Name", Guid.NewGuid());
+            var expectedSubcategory = new Subcategory {Name = "Name", CategoryId = Guid.NewGuid()};
```

Comment:
Add indents before and after curly braces, please
```suggestion
            var expectedSubcategory = new Subcategory { Name = "Name", CategoryId = Guid.NewGuid() };
```